docs/cli/cmd-options.mdx

```diff
@@ -518,6 +530,10 @@ Namespace Id.
 Disable folding.
 All Child Workflows within the set depth will be fetched and displayed.
 
+## no-json-shorthand-payloads
+
+Raw payload output, even if they are JSON.
```

Comment:
```suggestion
Raw payload output, even if the JSON option was used.
```

Reply:
Even if you don't like this wording, "they are" is not correct here with its pluralization